src/log.cpp

```diff
@@ -0,0 +1,101 @@
+#include "log.h"
+
+namespace bpftrace {
+
+static std::unordered_map<LogType, std::string> log_levels = {
+  { LogType::debug, "INFO" },
+  { LogType::semantic, "INFO" },
+  { LogType::semantic_warning, "WARNING" },
+  { LogType::semantic_error, "ERROR" },
+};
```

Comment:
I think an enum would be better here b/c std::unordered_map requires runtime initialization. And since it's not protected by a meyers singleton we open ourselves up to SIOF (which is more likely b/c logging can be expected to be used anywhere).

Reply:
I wanted this structure to give a string about log level, which is used by BPFtrace::log_with_location. I guess enum might not help with that. Maybe I would write a function which uses a switch statement to do the mapping. Not sure if that is good though. Btw, will taking away `static` solve the problem? Or it would be an issue as long as it's a global object.